google-cloud-datastore/src/main/java/com/google/cloud/datastore/DatastoreOptions.java

```diff
@@ -63,17 +69,31 @@ public ServiceRpc create(DatastoreOptions options) {
     }
   }
 
+  @Nonnull
+  com.google.cloud.datastore.telemetry.TraceUtil getTraceUtil() {
```

Comment:
please add some tests to `DatastoreOptionsTest`

Reply:
Done.

Reply:
Not seeing these, are we missing a commit?